Extension/src/LanguageServer/client.ts

```diff
@@ -1526,7 +1526,7 @@ export class DefaultClient implements Client {
                     const settings: CppSettings = new CppSettings();
                     if (changedSettings["formatting"]) {
                         const folderSettings: CppSettings = new CppSettings(this.RootUri);
-                        if (folderSettings.formattingEngine !== "Disabled") {
+                        if (folderSettings.formattingEngine !== "disabled") {
```

Comment:
You'll need to force the value we have to lowercase before doing the comparison.